src/js/utils/obj.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file obj.js
+ */
+
+/**
+ * Array-like iteration for objects.
+ *
+ * @param  {Object}   object
+ * @param  {Function} fn
+ *         A callback function which is called for each key in the object. It
+ *         receives the value and key as arguments.
+ */
+export function each(object, fn) {
```

Comment:
how much space would it take to bring in lodash here instead? (could provide some lodash code sharing between projects if it is not a lot more)